app/models/host/managed.rb

```diff
@@ -291,7 +291,8 @@ def handle_realm
   def diskLayout
     @host = self
     template = disk.blank? ? ptable.layout : disk
-    pxe_render(template.tr("\r", ''))
+    template_name = disk.blank? ? ptable.name : 'Custom disk layout'
+    unattended_render(template.tr("\r", template_name))
```

Comment:
Looks like you meant to pass template_name in as an argument to unattended_render, not to `tr`!